nx-cloud/values.yaml

```diff
@@ -1,11 +1,94 @@
-image:
-  tag: ''
+global:
+  imageRegistry: ''
+  imageTag: ''
 
 nxCloudAppURL: ''
 verboseLogging: false
 verboseMongoLogging: false
 enableMessageQueue: false
 
+frontend:
+  image:
+    registry: ''
+    repository: nxprivatecloud/nx-cloud-frontend
+    tag: ''
+    digest: ''
+    pullPolicy: Always
+  resources:
+    limits: {}
```

Comment:
Some OpenShift environments require limits to be specified. Thus, exposing them on configuration.